src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java

```diff
@@ -263,7 +266,7 @@ else if ( file != null && file.isDirectory() )
     }
 
     private Map<Artifact, Set<String>> buildUsedArtifacts( Map<Artifact, Set<String>> artifactClassMap,
-                                              Set<String> dependencyClasses )
+                                                           Set<String> dependencyClasses )
```

Comment:
Same here

Reply:
I remember that we do this everywhere although not being consistent

Reply:
so 
 - arguments for multiline method declaration - should be aligned
 - arguments for multiline method call - should be not aligned

corect?

Reply:
so 
 - arguments for multiline method declaration - should be aligned
 - arguments for multiline method call - should be not aligned

corect?

Reply:
@michael-o - what  formatting should be done ... I think that my proposition is correct